amazonses/amazonses-connector/amazonses-connector-1.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/amazonses/auth/AmazonSESVerifyEmailIdentityMediator.java

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  Copyright (c) 2005-2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
In the Copyright, we need not mention the range. Just the created year is enough, i.e., "Copyright (c) 2016"